deps/jerry/jerry-core/ecma/operations/ecma-exceptions.c

```diff
@@ -134,6 +136,42 @@ ecma_new_standard_error (ecma_standard_error_t error_type) /**< native error typ
 
   ((ecma_extended_object_t *) new_error_obj_p)->u.class_prop.class_id = LIT_MAGIC_STRING_ERROR_UL;
 
+  const char *frames_id_p = "_frames";
+
+  ecma_string_t *frames_str_p = ecma_new_ecma_string_from_utf8 ((const lit_utf8_byte_t *) frames_id_p, 7);
+
+  ecma_property_value_t *prop_value_p = ecma_create_named_data_property (new_error_obj_p,
+                                                                         frames_str_p,
+                                                                         ECMA_PROPERTY_CONFIGURABLE_WRITABLE,
+                                                                         NULL);
+  ecma_deref_ecma_string (frames_str_p);
+
+  uint32_t depth = 10;
+  // create frames
+  uint32_t *frames = malloc(sizeof(uint32_t) * depth);
+  memset(frames, 0, sizeof(uint32_t) * depth);
+  jcontext_get_backtrace_depth(frames, depth);
+
+  ecma_value_t frames_array_length_val = ecma_make_uint32_value (depth);
+  ecma_value_t frames_array = ecma_op_create_array_object (&frames_array_length_val, 1, true);
+  ecma_free_value (frames_array_length_val);
+  ecma_object_t *array_p = ecma_get_object_from_value (frames_array);
+
+  for (uint32_t i = 0; i < depth; ++i) {
```

Comment:
Let's follow the coding style of JerryScript, put the left-bracket `{` on the next new line.